pycalphad/io/database.py

```diff
@@ -33,6 +33,34 @@ def _to_tuple(lst):
     return tuple(_to_tuple(i) if isinstance(i, list) else i for i in lst)
 
 
+class Specie(object):
```

Comment:
The internet seems to say that "species" is spelled and pronounced the same way for the singular and plural case (like "sheep" or "deer"). I think we should compromise on clarity for the sake of grammatical correctness here.

Reply:
Well that's embarrassing... I'll fix it.